public/403.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>The change you wanted was rejected (422)</title>
```

Comment:
cancancan actually raises an exception in that case by default but according to the lines you added in the application_controller:
```ruby
rescue_from CanCan::AccessDenied do |exception|
  render :file => "#{Rails.root}/public/403.html", :status => 403, :layout => false
end
```
You'd get the 403 redirection effect but I'm talking about the content of the 403 page... The content here seems to be a response of 422... At least that's what the user would understand if he arrived here.